src/Menu/Menu.css

```diff
@@ -0,0 +1,57 @@
+.textbox {
+    margin-bottom: 20px;
+}
+
+.itemList {
+    list-style: none;
+    width: 280px;
+}
+.item {
```

Comment:
you could remove this class from each item in the item menu and just write
`.itemList > li { ... }`
instead